packages/application-components/src/components/portals-container/portals-container.tsx

```diff
@@ -21,27 +21,27 @@ type TPortalsContainerProps = {
    * The offset value for positioning the container from the top, when opened.
    * Usually this corresponds to the height of the header section.
    */
-  offsetTop: string;
+  offsetTop?: string;
   /**
    * The offset value for positioning the container from the left, when opened.
    * Usually this corresponds to the min width of the nav menu.
    */
-  offsetLeft: string;
+  offsetLeft?: string;
   /**
    * The offset value for positioning the container from the left, when opened.
    * The value is only applied when the `.body__menu-open` global class is added to the DOM.
    * Usually this corresponds to the width of the expanded nav menu.
    */
-  offsetLeftOnExpandedMenu: string;
+  offsetLeftOnExpandedMenu?: string;
   /**
    * The CSS selector to apply the `overflow: hidden` style to (preventing scrolling)
    * when a modal container is open.
    */
-  containerSelectorToPreventScrollingOnOpen: string;
+  containerSelectorToPreventScrollingOnOpen?: string;
```

Comment:
nit: Perhaps we could also include the default values in the jsdocs for these props (as it is done for the `zIndex` prop).